index.js

```diff
@@ -58,51 +67,109 @@ app.get('/', function (req, res) {
 
 app.post('/airdrop', (req, res) => {
     let ip = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
-
+    
     ip = ip.split(",").map(item => item.trim());
     ip = ip[0];
-
     let existingIP = historyStore
         .find({ ip: ip })
         .value()
+    let flag_mV = false;
+    let flag_mA = false;
+    let desmos_address = req.body.address;
+    let url1 = lcdAddress+"/validatorsets/latest"
+    let url2= lcdAddress+'/staking/delegators/'+desmos_address+'/validators'
+    let url3 = lcdAddress + '/bank/balances/'+desmos_address;
+    axios.all([
+        axios.get(url1),
+        axios.get(url2), 
+        axios.get(url3),
+    ]).then(axios.spread((response1, response2, response3) => {
+        let validators_list = response1.data.result.validators;
+        let assoc_validators = response2.data.result; 
+                        console.log("checking associated validator to the delegator address....")
+                        console.log(assoc_validators)      
+                        let assoc_validators_address = assoc_validators.map(assoc_validator => { return assoc_validator.operator_address; });
+                        console.log("Cross-linked Validators:....")
+                        console.log(assoc_validators_address)
+                        let matched_validators = validators_list.filter(lv => assoc_validators_address.includes(lv.address));
+                        console.log("Matched ones....")
+                        console.log(matched_validators)
+                        if(Array.isArray(matched_validators) && matched_validators.length){
+                        
+                            matched_validators.forEach( (mValidator,i) => {
+                                matched_validators[i].voting_power = parseInt(matched_validators[i].voting_power);
+                                console.log("stacking power: "+ matched_validators[i].voting_power);
+                                if(matched_validators[i].voting_power > 200){ 
+                                        flag_mV = true; 
+                                }
+                            }); 
+                        }
+                        else{
+                            console.log("the address not associated to validator or the validtor is jailed")
+                        }
+    
+        let  balances = response3.data.result;
+                        
+                        console.log("checking the balances with ....")
+                        console.log(balances);
+    
+                        balances.forEach((balance, i) => {
+                        if (balances[i] && balances[i].amount)
+                            balances[i].amount = parseFloat(balances[i].amount);
+                            console.log("the amount: "+ balances[i].amount)
+                            if(balances[i].amount > 10000000){
```

Comment:
@Sayeh-1337 one more thing. The parameters can be stored in environment variables so instead of hard coding here.